ksqldb-rest-app/src/main/java/io/confluent/ksql/rest/server/resources/streaming/PullQueryPublisher.java

```diff
@@ -99,6 +106,7 @@ public synchronized void subscribe(final Subscriber<Collection<StreamedRow>> sub
 
     PullQueryExecutionUtil.checkRateLimit(rateLimiter);
     final Decrementer decrementer = concurrencyLimiter.increment();
+    pullBandRateLimiter.allow(Time.SYSTEM.milliseconds());
```

Comment:
We're still trying to figure out how to get the bytes here, right?

If we just retain the qps and concurrency rate limiter across all endpoints, and we want to fall back on that here, then that might be sufficient, but if we wanted to lift the other rate limiting on the other endpoints in favor of bandwidth, then we couldn't just disable it because it would disable it here too.